

<launch>  
  
  

  <param name="file_name" value="/known_locs_serving_sim.yaml"/>

  <node pkg="hmm_navigation"  name="pumas_nav_actionserver" type="pumas_nav_actionserver.py"/>

  <node pkg="known_locations_tf_server"  name="known_locs" type="known_locs_node.py" output="screen" />
  <node pkg="object_classification"  name="objs_reco" type="classification_server.py"  />  
  <node pkg="segmentation"  name="segmentation" type="segment_node.py"  />  
  <node pkg="smach_viewer"  name="smach_viewer" type="smach_viewer.py"  />  
  <node pkg="human_detector"  name="human_detector" type="pointing_detector_server.py"/>  
  <node pkg="leg_finder"  name="leg_finder_node" type="leg_finder_node"/>  
  <node pkg="human_follower"  name="human_follower_node" type="human_follower_node"/>  
  

</launch>  
